bitcoin/sighash_multiple.rb

```diff
@@ -16,6 +16,8 @@ def hex_to_bin(s)
   [s].pack("H*")
 end
 
+output_index_error = "Output index error!".freeze
```

Comment:
Can we use all upper case letters for constants?

Reply:
ok